backend-nest/src/modules/platform/platform.controller.ts

```
@@ -7,53 +7,79 @@
 @CacheTTL(0)
+@ApiTags("platform")
 @Controller("platform")
 export class PlatformController {
     constructor(private readonly platformService: PlatformService) {}
 
     @Get("/all")
-    async getAllStops(
+    @ApiDescription({
+        summary: "List of all platforms",
+    })
+    @ApiQuery(metroOnlyQuery)
```

Biome

[error] This parameter is used before its declaration.

The parameter is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)